commit 5351d0e1e1c65e32f107f5540fc67b8c7a759710
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-03-12T21:36:17+00:00 (7 months ago)
    
    TOOLS-2186 sdcnode updates for Feburary 2019 node.js security releases
    Reviewed by: Orlando Vazquez <orlando@joyent.com>
    Approved by: Orlando Vazquez <orlando@joyent.com>

diff --git a/Makefile b/Makefile
index 4cad77c..63463e0 100644
--- a/Makefile
+++ b/Makefile
@@ -44,7 +44,7 @@ JSSTYLE_FLAGS	= -o indent=4,doxygen,unparenthesized-return=0
 SMF_MANIFESTS_IN = smf/manifests/cnapi.xml.in
 SMF_DTD		= $(REPO_ROOT)/tools/service_bundle.dtd.1
 
-NODE_PREBUILT_VERSION=v6.15.1
+NODE_PREBUILT_VERSION=v6.17.0
 NODE_PREBUILT_TAG=zone
 ifeq ($(shell uname -s),SunOS)
 	# Allow building on other than image sdc-minimal-multiarch-lts@15.4.1.
